docs/sphinx/source/forecasts.rst

```diff
@@ -447,7 +447,7 @@ for details.
     sandia_modules = retrieve_sam('sandiamod')
     cec_inverters = retrieve_sam('cecinverter')
     module = sandia_modules['Canadian_Solar_CS5P_220M___2009_']
-    inverter = cec_inverters['SMA_America__SC630CP_US_315V__CEC_2012_']
+    inverter = cec_inverters['SMA_America__SC630CP_US__with_ABB_EcoDry_Ultra_transformer_']
```

Comment:
Can you check how the pages render on readthedocs? You should be able to login to readthedocs and set it up so that this branch is built. If you've done that, thanks already.

Reply:
It's no longer necessary for contributors to create their own readthedocs account. Click on the `continuous-documentation/read-the-docs` check in the merge status box below.

Reply:
I've built this branch with readthedocs already and rendering looks good. Didn't realise there is `continuous-documentation/read-the-docs` below... that's so much easier. Thanks for pointing it out!